BitwardenKit/Core/Platform/Services/FlightRecorderStateService.swift

```diff
@@ -0,0 +1,24 @@
+// MARK: - FlightRecorderStateService
+
+/// A protocol for a service that provides state management functionality required by the flight recorder.
+///
+public protocol FlightRecorderStateService {
+    /// Returns the identifier for the currently active account.
+    ///
+    /// - Returns: The active account's unique identifier.
+    /// - Throws: An error if the active account cannot be determined.
+    ///
+    func getActiveAccountId() async throws -> String
```

Comment:
🤔 Should the new protocol `ActiveAccountStateProvider` be used instead of adding the same method here?